Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -83,6 +83,20 @@ namespace AzToolsFramework
                 return AZ::Failure(findCommonRootOutcome.TakeError());
             }
 
+            // order entities by their respective position within Entity Outliner
+            std::sort(
```

Comment:
nit, use `AZStd::sort`

Reply:
Because these all have a common ancestor there's also a lambda trick we can do here to make this a bit more efficient (we only have to retrieve the bus handler once instead of twice whatever our sorting complexity is):
```
            EditorEntitySortRequestBus::Event(
                commonRootEntityId, [&topLevelEntities](EditorEntitySortRequestBus::Events* sortRequests)
                {
                    AZStd::sort(
                        topLevelEntities.begin(), topLevelEntities.end(),
                        [&sortRequests](AZ::Entity* entity1, AZ::Entity* entity2)
                        {
                            return sortRequests->GetChildEntityIndex(entity1->GetId()) <
                                sortRequests->GetChildEntityIndex(entity2->GetId());
                        });
                });
```

Reply:
cool didn't know about that